pkg/didcomm/protocol/exchange/service/service_test.go

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
We need to improve coverage to 85

Reply:
Only lines I couldnt cover are lines in red below, since file is small it effects coverage.
Issue #113 will cover these scenarios.
![image](https://user-images.githubusercontent.com/29631944/63601613-c2c3ff80-c593-11e9-8769-53177a56730c.png)